index.js

```diff
@@ -83,6 +86,9 @@ function paginate(query, options, callback) {
 
     return Promise.props(promises)
         .then(function(data) {
+            if (data.count === null) {
+                data.count = data.docs.length
```

Comment:
If number of all docs is under 300(totalLimit), this data.count will be limit(default 10)?

<example>
actual data.count : 290,
limit : 30 

expected result : data.count = 290
this case : data.count = 10  (because docs contains 10 documents?)
right?

Reply:
This code removed.